app/src/main/java/com/eveningoutpost/dexdrip/SystemStatus.java

```diff
@@ -74,12 +77,24 @@ public void onNavigationDrawerItemSelected(int position) {
     private void set_current_values() {
         activeBluetoothDevice = ActiveBluetoothDevice.first();
         mBluetoothManager = (BluetoothManager) getSystemService(Context.BLUETOOTH_SERVICE);
+        setVersionName();
         setCollectionMethod();
         setCurrentDevice();
         setConnectionStatus();
         setNotes();
     }
 
+    private void setVersionName(){
+        String versionName = null;
+        try {
+            versionName = getPackageManager().getPackageInfo(getPackageName(), PackageManager.GET_META_DATA).versionName;
+            version_name_view.setText(versionName);
+        } catch (PackageManager.NameNotFoundException e) {
+            //e.printStackTrace();
+            Log.e(this.getClass().getSimpleName(),"PackageManager.NameNotFoundException:" + e.getMessage());
+        }
```

Comment:
When logging, please also add the exception itself as a third parameter.
Log.e(TAG, message, e)